README.md

```diff
@@ -25,8 +25,8 @@ This software is under development.
 <script>
   MathJax = {
     loader: {
-      load: ['[custom]/xypic.js'],
-      paths: {custom: 'https://cdn.jsdelivr.net/gh/sonoisa/XyJax-v3@3.0.1/build/'}
+      load: ['[xypic]'],
+      paths: {xypic: 'https://cdn.jsdelivr.net/gh/sonoisa/XyJax-v3@3.0.1/build/xypic'}
```

Comment:
While it is not necessary to have the `.js` in the `load` references, you probably do want to include the '.js' here, however, as this is an actual file reference.  It probably works without it, but it feels wrong to leave it off here.

Reply:
I agree that it looks a bit silly, but with a naked `'[xypic]'`, which I agree is the best option, it adds a .js extension no matter what - so you get a reference to xypic.js.js which doesn't load.

Reply:
OOPS!  Sorry, my bad.  I guess the filters that add the `.js` and that resolve the `[...]` are in the wrong order.  I'll look into that.  Thanks for letting me know.

Reply:
No worries! I should thank you for coming with the perfect build answer, you really made my day

Reply:
Glad to help.